kernel/kernel-keymanager-service/configure_start.sh

```
@@ -38,5 +38,13 @@ cd $DIR_NAME && chmod +x install.sh && sudo ./install.sh && chmod a-w install.sh
 echo "Installation complete"
 cd $work_dir
 
+echo "Downloading IAM adapter..."
+wget -q "${iam_adapter_url_env}" -O "${loader_path_env}/kernel-auth-adapter.jar"
```

Shellcheck (0.11.0)

[warning] iam_adapter_url_env is referenced but not assigned.

(SC2154)